packages/@lwc/integration-karma/test-hydration/mismatches/class-attr/static-same-with-static-parts/x/main/main.html

```diff
@@ -0,0 +1,4 @@
+<template>
+    <!-- presence of a dynamic attribute creates a static part -->
+    <p class="c1" style={s1}>text</p>
```

Comment:
Should we also test the reverse case, where the `style` is static but the `class` is dynamic?